From a82af933b2ee35e24140ee7fddf9a083596cda3f Mon Sep 17 00:00:00 2001
From: Dorian Stoll <dorian.stoll@tmsp.io>
Date: Mon, 29 Jan 2024 19:26:41 +0100
Subject: [PATCH] Pull in intel-vsc-firmware

This package contains the IPU3 firmware needed for camera support.
---
 redhat/kernel.spec.template | 1 +
 1 file changed, 1 insertion(+)

diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 4ece0fa3a56c..768316b2ec82 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -1028,6 +1028,7 @@ Requires(pre): %{kernel_prereq}\
 Requires(pre): %{initrd_prereq}\
 Requires(pre): ((linux-firmware >= 20150904-56.git6ebf5d57) if linux-firmware)\
 Recommends: linux-firmware\
+Recommends: intel-vsc-firmware\
 Recommends: iptsd\
 Recommends: %{package_name}-default-watchdog\
 Requires(preun): systemd >= 200\
-- 
2.43.0

